### ISSUE TITLE
We miss a `console.log()` call in the `Alert` component

### ISSUE THREAD

**Issue by ofrogon** [gsoft-inc/ov-igloo-ui#61]
A debugging `console.log()` was left in the `Alert` component in the `Alert.stories.tsx` file.

**Comment by ofrogon** [comment]
Here is the robot portrait of the suspect:

```js
    console.log(
      `Title: ${title}; Properties: ${alertProperties}; Index ${index}`
    );
```

**Comment by ofrogon** [comment]
Was fixed in PR #86
